pkg/cron_workflow.go

```diff
@@ -660,11 +645,75 @@ func (c *Client) GetCronWorkflowStatisticsForTemplates(workflowTemplates ...*Wor
 }
 
 func cronWorkflowColumns(extraColumns ...string) []string {
```

Comment:
Let's call this `getCronWorkflowColumns` so it matches the other, similarly named methods.

Here's one of the other ones,

```
func getWorkspaceStatusColumns(alias string, destination string, extraColumns ...string) []string {
	columns := []string{"phase", "started_at", "paused_at", "terminated_at"}
	return formatColumnSelect(columns, alias, destination, extraColumns...)
}
```

Reply:
Should this also be in `cron_workflow_types.go` like the other methods?

Reply:
@Vafilor Question about this one, it's being used by workflow_template.go?
So making it un-exported will cause an issue right?

Reply:
@aleksandrmelnikov no, that's in the same package.

Reply:
My bad, I thought it was upper case for some reason. Refactored.